src/lib/utils/customerUtils.ts

```diff
@@ -3,9 +3,9 @@ import { CustomerRecord, MeterReadingRecord, SiteRecord, TariffPlanRecord } from
 import { getCurrentPeriod, getDiffinPeriods, getPeriodFromDate } from '../../lib/moment/momentUtils';
 
 const isCurrentReading = (mr: MeterReadingRecord): boolean => {
-    const period: string = getPeriodFromDate(mr.date);
-    const currPer: string = getCurrentPeriod();
-    return period === currPer;
+  const period: string = getPeriodFromDate(mr.date);
+  const currPer: string = getCurrentPeriod();
+  return period === currPer;
```

Comment:
Nit: this feels like a lot of unnecessary shortening of variable names that doesn't help with readability

Reply:
i agree. I'm reverting this change to preserve blame (my linter just indented here)